python/dgl/heterograph_index.py

```diff
@@ -298,7 +300,7 @@ def has_nodes(self, ntype, vids):
             0-1 array indicating existence
         """
         vid_array = vids.todgltensor()
-        return utils.toindex(_CAPI_DGLHeteroHasVertices(self, int(ntype), vid_array))
+        return utils.toindex(_CAPI_DGLHeteroHasVertices(self, int(ntype), vid_array), self.dtype)
```

Comment:
When constructing `utils.Index` from a DGL array, we could omit the id type argument since the array contains the info.

Reply:
This is for sanity check to avoid mistake. Do you think it's necessary for DGL array to specify extra dtype when converting to `Index`?